lib/class-wp-theme-json-gutenberg.php

```diff
@@ -811,20 +811,46 @@ protected static function sanitize( $input, $valid_block_names, $valid_element_n
 	 *
 	 * @since 5.8.0
 	 * @since 6.1.0 Added append position.
+	 * @since 6.3.0 Deprecated append position parameter.
 	 *
-	 * @param string $selector  Original selector.
-	 * @param string $to_append Selector to append.
-	 * @param string $position  A position sub-selector should be appended. Default 'right'.
+	 * @param string $selector   Original selector.
+	 * @param string $to_append  Selector to append.
+	 * @param string $deprecated Deprecated parameter.
 	 * @return string The new selector.
 	 */
-	protected static function append_to_selector( $selector, $to_append, $position = 'right' ) {
+	protected static function append_to_selector( $selector, $to_append, $deprecated = 'right' ) {
```

Comment:
It is okay to do this because the method is protected right?

Reply:
I think so. Potentially we could even remove the parameter completely. The class is marked as private in its entirety, so probably no need to worry about backward compatibility on this protected method.

A quick plugin directory search indeed indicates that no plugin at all is extending `WP_Theme_JSON`: https://wpdirectory.net/search/01GZH0YP5WDAV9JF50VJCSYWZ2 The only instance is actually extending `WP_Theme_JSON_Resolver`.

So I think we should clean this up and remove the parameter entirely. I can make that update if there's agreement.